### PR TITLE
[FrameworkBundle] Move profiling collectors configuration to PHP

### PR DESCRIPTION
**Pull request by hvt**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | https://github.com/symfony/symfony/issues/37186
| License       | MIT

Moving profiling collectors configuration in framework bundle to PHP.

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    $container->services()
+        ->set('data_collector.config', ConfigDataCollector::class)
+            ->call('setKernel', [service('kernel')->ignoreOnInvalid()])
+            ->tag('data_collector', ['template' => '@WebProfiler/Collector/config.html.twig', 'id' => 'config', 'priority' => -255])
+
+        ->set('data_collector.request', RequestDataCollector::class)
+            ->tag('kernel.event_subscriber')
+            ->tag('data_collector', ['template' => '@WebProfiler/Collector/request.html.twig', 'id' => 'request', 'priority' => 335])
+
+        ->set('data_collector.request', AjaxDataCollector::class)
```
Comment (yceruto) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.php:35]: `data_collector.ajax`

Comment (hvt) [reply src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.php:35]: Aaargh, thx :].

Will the tests pick this up? Locally running `php ./phpunit src/Symfony/Bundle/FrameworkBundle` did not report any errors. (Or is it in another test?)

Comment (hvt) [reply src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.php]: Fixed it, was just wondering.

**Review by yceruto — APPROVED** [review]
LGTM, thanks!

**Review by Tobion — APPROVED** [review]

**Comment by Tobion** [comment]
Thank you @hvt.

**Pull request merged**